src/main/java/bisq/price/util/cryptoya/CryptoYaRate.java

```diff
@@ -0,0 +1,22 @@
+package bisq.price.util.cryptoya;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+public class CryptoYaRate {
```

Comment:
In the other packages its called `Ticker`. Better keep all the same, either rename the others to Rate or yours as Ticker.

Reply:
I'll change it to Ticker in my next commit for consistency. If we were to do renamings we can do a refactor PR later than down the road. Thanks!